pkg/storaged/client.js

```diff
@@ -741,6 +741,15 @@ function init_model(callback) {
         ).then(() => info);
     }
 
+    try {
+        client.anaconda = JSON.parse(window.localStorage.getItem("cockpit_anaconda"));
+    } catch {
+        console.warn("Can't parse cockpit_anaconda configuration as JSON");
+        client.anaconda = null;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231212-131008-f8bec949-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)